README.md

```diff
@@ -96,6 +96,9 @@ Possibly Asked Questions
 * Do I have to specify an index?
     * At Cerner: Yes. We use indexes to define ownership and access to data. Data sent to the default index will be rejected.
     * In general: No. Without specifying an index, you wind up in the 'main' index.
+* Can I use this cookbook to configure a Universal Forwarder for a host image?
+    * Yes, add `recipe[cerner_splunk::image_prep]` to the end of your run list.
+ 
```

Comment:
Nit extra line... 

But also could you squash your commits, and stop referencing the issue from your commit message? (as is stated in https://github.com/cerner/cerner_splunk/blob/stable/CONTRIBUTING.md#feature-develoment ) as you can see it's adding noise to the issue